.check_copyright_config.yaml

```diff
@@ -18,3 +18,4 @@ ignore:
   include:
     - src/target/*/ld/*.ld
     - example/ld/*.ld
+    - src/target/*/include/soc/uart_reg.h
```

Comment:
Doesn't `uart_reg.h` has proper license info? If not It is ok to modify the license part. We don't need to copy paste directly.

Reply:
Based on the change, this item should not go into the ignore list but a separate section defined for it accepting Apache 2.0 only. This comment applies only if we stick with Apache 2.0 option only.